### ISSUE TITLE
Filtering Loss Functions and Optimizers Parameters

### ISSUE THREAD

**Issue by Nada119** [mourra950/Siemens-System-Level-Modelling-of-ASDLA-Graduation-Project#60]
### What feature to add

Some Parameters in Loss Functions and Optimizers, which are in Miscellaneous parameters page, are not handled because of their data types. So we decided to Filter Loss Functions and Optimizers Parameters and choose the parameters which are essential to us and handle it.

### Milestone

Milestone 2

### What labels is that issue related to

Pytorch, Enhancement

**Comment by mourra950** [comment]
Successfully filtered in the AutoExtraction Module
